app/api.py

```diff
@@ -357,10 +357,34 @@ def create_ticket(ticket: TicketCreate) -> Ticket:
 def get_tickets():
     """Get all tickets.
 
-    This function is used to get all tickets.
+    This function is used to get all tickets with status open
```

Comment:
It's best here to allow adding filters to allow users to select the tickets they want to (open or closed, with the issue type they want to).
So that you have a single endpoint, instead of having 1 endpoint per tasks.

Reply:
Tanks for the review, I'll change that